upgrade/scripts/upgrade/prerequisites.sh

```diff
@@ -623,6 +667,26 @@ do_upgrade_csm_chart cray-spire sysmgmt.yaml
 do_upgrade_csm_chart cray-tapms-crd sysmgmt.yaml
 do_upgrade_csm_chart cray-tapms-operator sysmgmt.yaml
 
+# Running the rollout restart script to restart the required resources in istio-injection=enabled namespaces.
+state_name="RESTART_SERVICES_REFRESH_ISTIO"
+state_recorded=$(is_state_recorded "${state_name}" "$(hostname)")
+if [[ ${state_recorded} == "0" && $(hostname) == "${PRIMARY_NODE}" ]]; then
+  echo "====> ${state_name} ..." | tee -a "${LOG_FILE}"
+  {
+    bash rollout-restart.sh
```

Comment:
This will only work if prior to script execution, current directory is changed to script location. It is actually not. Also, `bash` is not required.
1. Please ensure that file mode on `rollout-restart.sh` script is 0755 and it has shebang line like this:
    
        #!/usr/bin/env bash
    
2. Invoke script as `${locOfScript}/rollout-restart.sh`